static/js/tippyjs.js

```diff
@@ -54,4 +54,8 @@ export function initialize() {
             instance.setContent(title);
         },
     });
+    delegate("body", {
```

Comment:
You don't need to create a new delegate here, you can simply use `tippy-zulip-tooltip` everywhere.

Reply:
Got it. Thanks.